ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_installation.py::TestInstallCA::test_certmonger_reads_token_HSM
         template: *ci-master-latest
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: *master_2repl_1client
```

Comment:
Are two replicas really necessary here?

Reply:
Yes. The other tests in the class requires 2 replicas.